### PR TITLE
document an example how to use a static PersistentVolume

### PR DESCRIPTION
**Pull request by aep**
thanks to @alicefr for helping me figure this out.
maybe the example is useful to someone else.

### CODE REVIEW

File: docs/virtual_machines/disks_and_volumes.md [hunk 1]
```diff
@@ -412,6 +412,56 @@ spec:
         claimName: mypvc
 ```
 
+
+#### block device from host
```
Comment (alicefr) [inline docs/virtual_machines/disks_and_volumes.md:416]: I would call this `Statically provisioned block PVCs`

File: docs/virtual_machines/disks_and_volumes.md [hunk 2]
```diff
@@ -412,6 +412,56 @@ spec:
...
+
```
Comment (alicefr) [inline docs/virtual_machines/disks_and_volumes.md:417]: Here, are missing a short introduction why we need statically PVC. You could mentioned, your use-case to share a disk from the host to the VM

File: docs/virtual_machines/disks_and_volumes.md [hunk 3]
```diff
@@ -412,6 +412,56 @@ spec:
...
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - my-node
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 100Gi
+  local:
+    path: /dev/sdb
```
Comment (alicefr) [inline docs/virtual_machines/disks_and_volumes.md:462]: Please, split the storage class and PV creation to the PVCs definition. We should make clear here that PVs ans SCs can only be created by the cluster-admin and not by regular users. Regular users can only get storage classes and create PVCs

File: docs/virtual_machines/disks_and_volumes.md [hunk 4]
```diff
@@ -412,6 +412,62 @@ spec:
...
+  resources:
+    requests:
+      storage: 100Gi
+```
+
+can claim a PersistentVolume pre-created by a cluster admin like so:
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```
Comment (mhenriks) [inline docs/virtual_machines/disks_and_volumes.md:443]: I haven't tried this in awhile but am surprised that the `StorageClass` resource is necessary.  What happens if it is omitted?

Comment (aep) [reply docs/virtual_machines/disks_and_volumes.md:443]: it'll say it doesnt know the storageclass

**Pull request merged**